src/components/shared/SectionStepInfo/SectionStepInfo.module.scss

```diff
@@ -0,0 +1,10 @@
+.sectionWrapper {
+  .section {
+    font-size: 16px;
+    color: #5f6166;
+  }
+  .description {
+    font: 18px;
+    color: #54565b;
+  }
+}
```

Comment:
Not sure these classes are needed, see progress indicator: https://designsystem.wmnetwork.co.uk/components/form-elements/